app/tests/Feature/Http/VotingLists/JsonTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+use App\Enums\CountryEnum;
+use App\Enums\VoteResultEnum;
+use App\Group;
+use App\Vote;
+use App\Member;
+use App\VotingList;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Carbon;
+
+uses(Tests\TestCase::class, RefreshDatabase::class);
+
+beforeEach(function () {
+    $date = new Carbon('2021-05-23');
+    $display_title = 'Budged 2020';
+
+    $greens = Group::factory([
+        'code' => 'GREENS',
+        'name' => 'Greens/European Free Alliance',
+        'abbreviation' => 'Greens/EFA',
+    ])->create();
+
+    $this->greensFor = Member::factory([
+        'first_name' => 'Jane',
+        'last_name' => 'DOE',
+        'country' => CountryEnum::NL(),
+    ])->activeAt($date, $greens)->count(1);
+
+    $this->votingList = VotingList::factory([
+        'date' => $date,
+        'vote_id' => Vote::factory([
+            'result' => VoteResultEnum::REJECTED(),
+        ]),
+        'description' => 'Budged 2020',
+        ])
+        ->withMembers('FOR', $this->greensFor)
+        ->create();
+});
+
+it('creates a JSON with one row per member', function () {
+    $response = $this->get("/votes/{$this->votingList->id}.json");
+
+    $decoded = json_decode($response->content(), true);
```

Comment:
There’s a Laravel helper method for this as well. You can simply do the following:

```suggestion
    $response = $this->get("/votes/{$this->votingList->id}.json")->json();
```